### DISCUSSION TITLE
#Introductions

### DISCUSSION THREAD

**Discussion by snotskie** [snotskie/EpistemicNetworkAnalysis.jl#7] [Show and tell]
Hey everyone!

I'm Mariah A. Knowles, I'm a sixth year PhD candidate in the UW-Madison iSchool, and Data Science Facilitator for the Data Science Hub

I created EpistemicNetworkAnalysis.jl to help explore different ways to rotate ENA models

I've since used it in:

- [Hiearchical Epistemic Network Analysis (poster)](https://www.qesoc.org/images/pdf/ICQE20_Proceedings_Supplement_Final_web.pdf#page=37)
- [Telling Stories of Transitions (paper)](https://link.springer.com/chapter/10.1007/978-3-030-93859-8_8)
- [Code-wise ENA: A Sequential Representation of Discourse (poster)](https://www.qesoc.org/images/pdf/ICQE22_proceedings_supplement.pdf#page=61)
- [Multiclass Rotations in Epistemic Network Analysis (paper)](https://link.springer.com/chapter/10.1007/978-3-031-31726-2_5)
- [How Can We Co-design Learning Analytics for Game-Based Assessment: ENA Analysis (paper)](https://link.springer.com/chapter/10.1007/978-3-031-31726-2_15)

I chose to develop this package in Julia instead of extending the original R package because (a) rewriting the package from scratch was tremendously useful for learning how ENA works, (b) Julia's typesystem allowed me to solve API design issues with ENA's composed type tree, and (c) I've been using Julia since its early days and find it more expressive than R for the kind of work I do

I've been working on this package for the past three years, and since then I've grown so much as a programmer and researcher. I'm so so so excited to be sharing it with you and to hear what y'all come up with using it!

And if you're attending ICQE23, come check out my workshop where I'll be introducing this package and talking about how to choose the right rotation for the job!

**Comment by HumblxScience** [comment]
Hello Mariah! Looking forward to meeting everyone and this workshop at ICQE23.

I am a PhD student in the Pepperdine University Graduate School of Education and Psychology, Global Leadership and Change program. My area of study is using QE and ENA to examine Artificial Intelligence (AI) Governance and Human Factors Science/Engineering.

Respectfully,
Jodi Masters-Gonzales
